crates/kornia-io/src/jpeg.rs

```diff
@@ -0,0 +1,201 @@
+use crate::error::IoError;
+use image::codecs::jpeg::JpegEncoder;
+use image::{ExtendedColorType, ImageEncoder};
```

Comment:
what about the encoder? maybe this one https://github.com/vstroebel/jpeg-encoder